src/VisualStudio/Core/Def/Implementation/Library/ObjectBrowser/AbstractObjectBrowserLibraryManager.cs

```diff
@@ -508,5 +512,51 @@ protected override bool TryExec(Guid commandGroup, uint commandId)
 
             return false;
         }
+
+        private IStreamingFindUsagesPresenter GetStreamingPresenter()
+        {
+            try
+            {
+                return _streamingPresenters.FirstOrDefault()?.Value;
+            }
+            catch
```

Comment:
This shouldn't be needed.  It was a holdover to the days where our code and the editor's code for FAR were moving around the divisoin through different RIs/FIs.  Now that everything has flowed everywhere, i can remove this.